test/parallel/test-argv0-property.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+assert.ok(process.hasOwnProperty('argv0'), 'argv0 property exists');
+assert.throws(() => {
+  process.argv0 = 'anything';
+}, /read only/);
+assert.doesNotThrow(() => {
+  delete process.argv0;
+  Object.defineProperty(process, 'argv0', {value: 'some-test-value'});
+}, Error, 'argv0 is configurable');
```

Comment:
Is the `Error` needed (I hate the internals of `assert.doesNotThrow` I'm not going to look 🙈), could it be `null` instead?